source/release-notes.txt

```diff
@@ -11,6 +11,36 @@ Release Notes
    :class: twocols
 
 
+|compass| 1.43.2
+----------------
+
+*Released June 25, 2024*
+
+New Features:
+
+- Shows tool tip when a query or aggregation is generated without
+  content (:issue:`COMPASS-7837`).
+- Confirmation dialog when quitting Compass (:issue:`COMPASS-6435`).
+- Expands options when applied from a query history that has options.
+
+Bug Fixes:
+
+- Regular expression and number query history fix
+  (:issue:`COMPASS-7215`, :issue:`COMPASS-7008`).
+- Keeps listeners for insert document validity on document view
+  (:issue:`COMPASS-3246`).
+- Hides inaccurate collection statistics for timeseries
+  (:issue:`COMPASS-6712`).
+- Removes 'Preview' label from OIDC (:issue:`COMPASS-7666`).
+- Uses system ca certificates in Atlas requests and OIDC
+  (:issue:`COMPASS-7950`).
+- Hides edit view button in read only mode (:issue:`COMPASS-7688`).
+- Updates line numbers to be unselectable (:issue:`COMPASS-7941`).
+- Sidebar tab fix.
+
+`Full Changelog available on GitHub
+<https://github.com/mongodb-js/compass/compare/v1.43.1...v1.43.2>`__
+
```

Comment:
Can you please add an "e" to June in the previous release. We abbreviate months to 3 letters in the Server release notes, but here it just looks misspelled.

Reply:
Done.